### DISCUSSION TITLE
Renovate 36 on developer.mend.io

### DISCUSSION THREAD

**Discussion by remal** [renovatebot/renovate#23222] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Hello,

What is the plan on upgrading Renovate to v36 on developer.mend.io?

I'm asking becase v36 introduced breaking changes in config, so a lot of repositories that relied on config sections like `java`/`js`/etc can be broken.

I already prepared almost all of my repos, but I think a lot of people will get their configs broken.

Also, the issue is that sections like `java` are silently ignored. I think it makes sense to at least log it (see https://github.com/renovatebot/renovate/discussions/23192) or report as platform (i.e. GitHub, GitLab, etc) issue.

**Comment by rarkins** [comment, marked as answer]
We are currently rolling it out selectively for testing. If you'd like your orgs enabled first then please let us know which ones. Otherwise it could be Sunday when we enable it by default

**Reply by remal** [reply]
remal (this account) 
remal-gradle-plugins
remal-github-actions

Thanks!

**Reply by nabeelsaabna** [reply]
done

**Reply by jtoar** [reply]
@rarkins if [redwoodjs/redwood](https://github.com/redwoodjs/redwood) could be enabled that'd be great. I just went through the release notes and migrated to the new preset name, but it looks like the version is still 35, so I got this error: https://github.com/redwoodjs/redwood/issues/8860

**Reply by nabeelsaabna** [reply]
@jtoar done

**Comment by viceice** [comment]
we already fixed the config error and migrate those to package rules.
so it should still work
